show/main.py

```diff
@@ -2146,5 +2146,21 @@ def tablelize(keys, data, enable_table_keys, prefix):
     click.echo(tabulate(tablelize(keys, data, enable_table_keys, prefix), header))
     state_db.close(state_db.STATE_DB)
 
+#
+# show features
+#
+
+@cli.command('features')
+def features():
+    """Show status of optional features"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    header = ['Feature', 'Status']
+    body = []
+    status_data = config_db.get_table('FEATURES')
```

Comment:
Naming convention for tables in SONiC DB is singular form. Can you change this to `FEATURE`?

Reply:
updated